config/core/rootfs-configs.yaml

```diff
@@ -1,4 +1,21 @@
 rootfs_configs:
+  bullseye-libcamera:
+    rootfs_type: debos
+    debian_release: bullseye
+    arch_list:
+      - amd64
+      - arm64
+      - armhf
+    extra_packages:
+      - ca-certificates
+      - libevent-dev
+    extra_packages_remove:
+      - bash
+      - e2fslibs
+      - e2fsprogs
+    script: "scripts/bullseye-libcamera.sh"
```

Comment:
This looks fine.  If you have any issues with the rootfs, maybe drop the `script` line initially to check whether the basic `bullseye` rootfs image boots successfully.  That would help you determine if any issues are due to the Debian rootfs or the libcamera build script.